fire_atlas_editor/lib/screens/editor_screen/widgets/toolbar.dart

```diff
@@ -83,11 +79,12 @@ class Toolbar extends StatelessWidget {
                         ),
                       );
                     }),
-                FIconButton(
-                    iconData: Icons.get_app,
-                    onPress: () {
-                      _launchURL(slice.currentAtlas!);
-                    }),
+                if (kIsWeb)
+                  FIconButton(
+                      iconData: Icons.get_app,
+                      onPress: () {
+                        _launchURL(slice.currentAtlas!);
+                      }),
```

Comment:
trailing comma

Reply:
This is all messed up on the project, I will do a follow up to configure a better linter in the future when we have our flame-lint